sdk/python/feast/cli.py

```diff
@@ -120,15 +120,52 @@ def feature_set():
     pass
 
 
+def _get_labels_dict(label_str: str):
```

Comment:
We should start adding comments everywhere, this private method is no different. Please add Google Docstring (Intellij can generate).